test/e2e/cloud/gcp/resize_nodes.go

```diff
@@ -19,6 +19,7 @@ package gcp
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
I'm surprised your IDE does not arrange this imports, usually IDEs use `goimports` or `gopols` and this should go in the block below , this block is for the std libraries

Reply:
I enabled it now! should be fixed